### DISCUSSION TITLE
Image sizes optimised on generate?

### DISCUSSION THREAD

**Discussion by procrates** [nuxt/image#752] [Q&A]
Hi! Testing out nuxt-image with static generation using git lfs with netlify large media (tried vercel, no support for lfs) and image is rendring just fine.

What confuses me is the image size after deployment to netlify. The src-set is generating with all the sizes, but every image is the same size as the original.

So i tested the example provided with the same result.

Is this module supposed to optimise images on generate or should it be done manually/programatic before generate?

Thanks for answers.

**Comment by DavidDeSloovere** [comment]
What is you config and how are you referencing the images (ie URL)?
